src/test/java/io/aiven/kafka/connect/common/grouper/RecordGrouperFactoryTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.grouper;
+
+import io.aiven.kafka.connect.common.templating.Template;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+final class RecordGrouperFactoryTest {
+
+
+    @Test
+    void keyOnly() {
+        final Template filenameTemplate = Template.of("{{key}}");
+        final String grType = RecordGrouperFactory.resolveRecordGrouperType(filenameTemplate);
+        assertEquals(RecordGrouperFactory.KEY_RECORD, grType);
```

Comment:
Should we use AssertJ everywhere?

Reply:
Hi @stephen-harris, sorry for hijacking the workflow but could you please rebase onto main and fix the assertions?

Reply:
+1, thanks @AnatolyPopov, missed this in the review. 
@stephen-harris If you rebase your PR, this will be validated by checkstyle based on #192

Reply:
@AnatolyPopov @jeqo Rebased and updated to use AssertJ